pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/parse/abstraction/mapping/MedicalRecordAbstractionMappingGenerator.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsm.model.elastic.export.parse.abstraction.mapping;
```

Comment:
I believe all this generator classes belongs to `elastic.export.generate` package and you would not have to change access level of `BaseParser` methods.